test/e2e/proxy.go

```diff
@@ -155,9 +155,6 @@ func proxyContext(version string) {
 					if e, a := val, string(body); e != a {
 						recordError(fmt.Sprintf("%v: path %v: wanted %v, got %v", i, path, e, a))
 					}
-					if d > 15*time.Second {
-						recordError(fmt.Sprintf("%v: path %v took %v > 15s", i, path, d))
-					}
 				}(i, path, val)
 				time.Sleep(150 * time.Millisecond)
```

Comment:
it was flaky because default QPS is 5, and sleep time was less (150ms < 200ms), so if enough requests happened, later ones became more and more delayed